cafebabel/static/css/styles.css

```diff
@@ -1721,34 +1748,64 @@ textarea#body {
   font-family: serif;
 }
 
-.article-detail .tags-list a,
-.article-page .tags-list a {
+.tags-list a {
   position: relative;
-  font-size: 1.6rem;
-  padding: 0.5rem 1rem;
+  font-size: 1rem;
+  padding: 0.3rem 0.5rem;
   background-color: #eeeeee;
   display: inline-block;
   margin: 0 1rem 1rem 0;
   color: #696969;
 }
 
-.article-detail .tags-list a:after,
-.article-page .tags-list a:after {
+.tags-list a:after {
   content: none;
 }
 
-.article-detail .tags-list a:before,
-.article-page .tags-list a:before {
+.tags-list a:before {
   content: '#';
 }
 
-.article-detail .tags-list a:hover,
-.article-page .tags-list a:hover {
+.tags-list a.impact,
+.tags-list a.experience,
+.tags-list a.raw,
+.tags-list a.creative {
+  color: #00283c;
+}
+
+.tags-list a.impact:before,
+.tags-list a.experience:before,
+.tags-list a.raw:before,
+.tags-list a.creative:before {
+  font-family: 'icomoon';
+  padding-right: 0.4rem;
+}
+
+a.impact:before {
+  content: '\e904';
+}
+
+a.experience:before {
+  content: '\e903';
+}
+
+.tags-list a.experience:before {
```

Comment:
Not very proud of if, but since the eye 👀 picto is a bit wider (and less high), it gets bigger than the others with the same font-size. That's why I reduced the font-size juste for this one. I'd like to keep it like that, cause I like the way _icomoon_ works. But if you think it's a shame, then i can switch for separate svg icon pseudo-elements.

Reply:
Alright, are the `eot`, `ttf` and `woff` files generated from the custom-made `svg` one?

Reply:
oui !

Reply:
Where is documented the process, how do you do that? Would you mind amending the README file to add the recipe? :)

Reply:
I created a set in my _icomoon_ account. Then I chose icon that are already created by them AND added more icons from my own (created on illustrator then exported in svg). Then I created a font out of it. Each icon has a css unicode so that we can use them as we want ;-)